client/src/components/login/login-form/LoginForm.js

```diff
@@ -20,7 +33,7 @@ export default function LoginForm({history}) {
   }
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)}>
+    <form onSubmit={handleSubmit(attemptLogin)}>
```

Comment:
Changed the name of our onSubmit method to attemptLogin so that it can provide more context as to what is happening on submission of the form.

This line now reads as "Form, on submit, attempt login". +1 Informative!